pkg/interface/publish/src/js/components/root.js

```diff
@@ -78,20 +78,39 @@ export class Root extends Component {
           let ship = props.match.params.ship || "";
           let notebook = props.match.params.notebook || "";
 
-          return (
-            <Skeleton
-            popout={false}
-            active={"rightPanel"}
-            rightPanelHide={false}
-            sidebarShown={true}
-            notebooks={state.notebooks}>
-              <Notebook
-              notebooks={state.notebooks}
-              view={view}
-              ship={ship}
-              book={notebook}/>
-            </Skeleton>
-          )
+          if (view === "new") {
```

Comment:
Ooh, I like this a lot — but I don't think there's a mount for this param on the back-end, so while navigating to the "new" view will work fine, going directly to it won't work. Should be added I suppose.

Reply:
Just tested and it does work, so nvm! Cool cool cool. Let me look at testing...